### DISCUSSION TITLE
How do you configure resource limits

### DISCUSSION THREAD

**Discussion by fzwu** [kubernetes-client/csharp#1251] [Q&A] [answered]
at method await client.AppsV1.CreateNamespacedDeploymentAsync(createdeployment, vo.NameSpaces);
if configure  ResourceQuantity("100m") then return Operation returned an invalid status code 'UnprocessableEntity' 
when configure  ResourceQuantity("100") that's OK?

But here we need to configure the units,so how do I do that

**Comment by tg123** [comment, marked as answer]
could you please show me the code 
`ResourceQuantity("100m")` looked good to me

current sdk will throw detailed error from server

**Reply by fzwu** [reply]
I have solved the problem.
